TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utilities/Scissors.java

```diff
@@ -0,0 +1,35 @@
+package org.firstinspires.ftc.teamcode.utilities;
+
+import com.acmerobotics.dashboard.config.Config;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+
+@Config
+public class Scissors {
+
+    private Servo scissorsServo;
+    private Servo secondScissorsServo;
+
+    public static double fullyExtendedPosition = 1.0; // fully extended position
+    public static double loadingPosition = 0.0; // transferring/loading position
+    public static double idlePosition = 0.5; // not doing anything position
+
+    public Scissors(HardwareMap hardwareMap) {
+        this.scissorsServo = hardwareMap.servo.get(CONFIG.scissorsServo);
+        this.secondScissorsServo = hardwareMap.servo.get(CONFIG.secondScissorsServo);
+    }
+
+    public void moveToFullyExtended() {
+        scissorsServo.setPosition(fullyExtendedPosition);
+        secondScissorsServo.setPosition(loadingPosition);
```

Comment:
do like 1 - fullyExtendedPosition because it makes the most sense to a reader (one minus x being the inverse idk) also do this for all the functions